galasa-parent/dev.galasa.framework.api.common/src/main/java/dev/galasa/framework/api/common/resources/CPSNamespace.java

```diff
@@ -81,7 +81,7 @@ public CPSProperty getPropertyFromStore(String propertyName) throws Configuratio
         if (visibility != Visibility.HIDDEN) {
             GalasaPropertyName propName = new GalasaPropertyName(this.name, propertyName);
             prop = new CPSProperty(this.propertyStore, this, propName);
-            prop.LoadValueFromStore();
+            prop.loadValueFromStore();
```

Comment:
Why is this a public method ?

And 
```
public CPSProperty getProperty(String propertyName) throws ConfigurationPropertyStoreException {
        CPSProperty prop = getPropertyFromStore(propertyName);
            if (!prop.existsInStore()) {
                prop = null;
            }
            return prop;
        }
```
Why do we need that ? Why can't you create a property which doesn't exist in the store yet, and then tell the property to load/save ?

Reply:
Don't think I like this method. Not easy to see what's using it.